initfiles/componentfiles/configschema/templates/AddVaults.json

```diff
@@ -0,0 +1,102 @@
+{
+    "name": "Add Vault Feature",
+    "description": "Add necessary keys with empty values for vault feature",
+    "type": "modification",
+    "operations": [
+        {
+            "action": "delete",
+            "target_path": "/Environment/Software/vaults",
+            "data": {
+                "error_if_not_found": false
+            }
+        },
+        {
+            "action": "create",
+            "target_path": "/Environment/Software",
+            "data": {
+                "node_type": "vaults",
+                "save": {
+                    "name" : "vaults_node"
+                }
+            }
+        },
+        {
+            "action": "create",
+            "description": "Create the git element in Environment/Software/vaults",
+            "target_nodeid": "{{vaults_node}}",
+            "data": {
+                "node_type": "git",
+                "attributes": [
+                    {
+                        "name": "name",
+                        "value": "git_vault"
+                    },
+                    {
+                        "name": "url",
+                        "value": "default"
```

Comment:
Having it not empty as a workaround is fine, but I wouldn't use the term default because to me that makes it sound like the back end will just use some default value.  Something saying they need to fill in the value would be better.

Reply:
This has been modified to 

url : required_insert_url